Jasmine/Common/Extensions/Array+Extensions.swift

```diff
@@ -0,0 +1,21 @@
+extension Array where Element: Equatable {
+    /// Returns an array with the contents of this sequence, shuffled.
+    func isSubsequenceOf(_ other: [Element]) -> Bool {
+        for otherIndex in 0..<(other.count - count) {
+            var inThisRow = true
+
+            for selfIndex in 0..<count {
+                if self[selfIndex + otherIndex] != other[otherIndex] {
+                    inThisRow = false
+                    break
+                }
+            }
+
+            if inThisRow {
+                return true
```

Comment:
this is a common anti-pattern, should refactor the body of the for loop into separate func instead of using a boolean flag.